package.json

```diff
@@ -19,47 +19,72 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=10.0.0",
+    "node": "^10.13 || ^12.13",
```

Comment:
We only support Node 10 and 12 (starting from the minor version that made them LTS)